packages/framework/esm-styleguide/src/workspaces/container/workspace-renderer.component.tsx

```diff
@@ -35,6 +37,7 @@ export function WorkspaceRenderer({ workspace, additionalPropsFromPage }: Worksp
         setTitle: workspace.setTitle,
         ...additionalPropsFromPage,
         ...workspace.additionalProps,
+        ...workspaceFamilyState,
```

Comment:
```suggestion
        ...workspaceFamilyState,
        ...workspace.additionalProps,
```

The `additionalProps` that come directly from `launchWorkspace` are more specific, so I'd have those override the family state.